src/main/java/com/commercetools/sync/categories/CategorySync.java

```diff
@@ -0,0 +1,194 @@
+     */
+    private void syncCategories(@Nonnull final Category oldCategory, @Nonnull final CategoryDraft newCategory) {
+        final List<UpdateAction<Category>> updateActions =
+                CategorySyncUtils.buildActions(oldCategory, newCategory, getSyncOptions());
+        if (!updateActions.isEmpty()) {
+            updateCategory(oldCategory, updateActions);
+        }
+    }
+
+    /**
+     * Given a {@link Category} and a {@link List} of {@link UpdateAction} elements, this method issues a blocking request
+     * to the CTP project defined by the client configuration stored in the {@code syncOptions} instance of this class to
+     * update the specified category with this list of update actions.
+     * <p>
+     * The {@code statistics} instance is updated accordingly to whether the CTP request was carried out successfully or not.
+     * If an exception was thrown on executing the request to CTP,
+     * the optional error callback specified in the {@code syncOptions} is called.
+     *
+     * @param category      the category to update.
+     * @param updateActions the list of update actions to update the category with.
+     */
+    void updateCategory(@Nonnull final Category category,
+                        @Nonnull final List<UpdateAction<Category>> updateActions) {
+        try {
+            getSyncOptions().getCategoryService().updateCategory(category, updateActions);
+            getStatistics().incrementUpdated();
+        } catch (SphereException e) {
+            failSync(format("Failed to update category with id" +
+                            " '%s' in CTP project with key '%s",
+                    category.getId(), getSyncOptions().getClientConfig().getProjectKey()), e);
+        }
+    }
+
+    /**
+     * Given a reason message as {@link String} and {@link Throwable} exception, this method calls the optional error
+     * callback specified in the {@code syncOptions} and updates the {@code statistics} instance by incrementing the
+     * total number of failed resources to sync.
+     *
+     * @param reason    the reason of failure.
+     * @param exception the exception that occurred, if any.
+     */
+    private void failSync(@Nonnull final String reason, @Nullable final Throwable exception) {
+        getSyncOptions().callUpdateActionErrorCallBack(reason, exception);
+        getStatistics().incrementFailed();
+    }
+
+    /**
+     * Builds a JSON String that represents the fields of the {@code statistics} instance.
+     *
+     * @return a JSON String that represents the fields of the {@code statistics} instance.
+     */
+    @Override
+    @Nonnull
+    public String getSummary() {
+        return getStatisticsAsJSONString(getStatistics());
+    }
+
+    @Override
+    public <T> void sync(@Nonnull final List<T> categories) {
+        //TODO: SEE GITHUB ISSUE#12
+    }
+
+    CategorySyncOptions getSyncOptions() {
+        return syncOptions;
+    }
+
+    CategorySyncStatistics getStatistics() {
```

Comment:
It was actually meant as a package-private getter. The other one was `getSummary` which was public. But that was confusing regardless. It is fixed here: f0d240f